pywebpush/tests/test_webpush.py

```diff
@@ -106,7 +106,7 @@ def test_send(self, mock_post):
         ok_('encryption' in pheaders)
         eq_(pheaders.get('AUTHENTICATION'), headers.get('Authentication'))
         ckey = pheaders.get('crypto-key')
-        ok_('pre-existing,' in ckey)
+        ok_('pre-existing' in ckey)
```

Comment:
Kinda debating that. By checking `pre-existing` with no punctuation, i'm futureproofing the test. The problem is that due to a bit of fuzziness around some of the specs, some implementations use "," to separate key groups where others only use ";" and don't support key groups. Really, what I want to check is the pre-existing string is present. I could do clever regex tricks, but that also seems like overkill.